lib/HueSensor.js

```diff
@@ -1077,7 +1109,7 @@ function HueSensor (accessory, id, obj) {
         key: 'humidity',
         name: 'humidity',
         unit: '%',
-        history: 'weather',
+        history: this.obj.modelid === 'lumi.airmonitor.acn01' ? 'room2' : 'weather',
```

Comment:
That'll work for now, but won't scale well when adding more Eve-Room-like sensors in the future.